.github/workflows/ci_workflows.yml

```
@@ -3,7 +3,7 @@ name: CI
 on:
   push:
     branches:
-    - main
+    - v4.3.x
     tags:
     - '*'
   pull_request:
```

Yes, also this:

```suggestion
  pull_request:
  - v4.3.x
```

To be consistent with:

https://github.com/astropy/astropy/blob/5bce144c6c3e8697496eeab87e689989f1dc5549/.github/workflows/ci_workflows.yml#L3-L9